src/App.tsx

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+
+export const App: React.FC = () => {
+  return (
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <header className="todoapp__header">
+          {/* this button should have `active` class only if all todos are completed */}
+          <button
+            type="button"
+            className="todoapp__toggle-all active"
```

Comment:
The `className` for `ToggleAllButton` includes `active` by default. Ensure this class is conditionally applied based on whether all todos are completed, as per the comment on line 11.